blivet/__init__.py

```diff
@@ -2007,7 +2007,10 @@ def _updateCustomStorageKSData(self):
                  BTRFSDevice: ("BTRFSData", "btrfs")}
 
         # make a list of ancestors of all used devices
-        devices = list(set(a for d in list(self.mountpoints.values()) + self.swaps
+        bootLoaderDevices = []
+        if self.bootLoaderDevice is not None:
+            bootLoaderDevices.append(self.bootLoaderDevice)
+        devices = list(set(a for d in list(self.mountpoints.values()) + self.swaps + bootLoaderDevices
                                 for a in d.ancestors))
 
```

Comment:
Please change the indentation of the last line, it's floating somewhere in the middle of the line.